packages/perseus/src/styles/articles.less

```diff
@@ -418,7 +421,8 @@
 
     .math(@blockMathTextSize; @blockMathLineHeight;
           @inlineMathTextSize; @inlineMathLineHeight) {
-        .katex, mjx-container {
+        .katex,
+        mjx-container {
```

Comment:
I edited this file recently and it apparently wasn't auto-formatted. Same with `katex-mhchem/parser.js` above. How are we supposed to format these files, or prevent bad formatting from getting merged to `main`? Does `yarn lint` not run prettier?

Reply:
That's a good call Ben.  Originally, we only had Prettier running over JS source files, but that wasn't encoded in our `.prettierignore` so IDEs looking up our prettier config thought they could format a broader set of files. 

I've fixed things up so that our format script considers the full set of files. 

I've also cleaned up any violations.